src/providers/humidity.toit

```diff
@@ -24,9 +24,25 @@ abstract class HumidityProvider extends services.ServiceProvider implements serv
     unreachable
 
 class HumidityProvider_ extends HumidityProvider:
```

Comment:
If we're typically going to subclass anyway, then maybe using virtual methods rather than lambdas for the open/close stuff would be better?

Reply:
In later PRs we only use lambdas. Subclassing will disappear (at least in most cases).